client/my-sites/plugins/plugins-search-results-page/index.jsx

```diff
@@ -18,61 +34,17 @@ function isNotBlocked( plugin ) {
 	return PLUGIN_SLUGS_BLOCKLIST.indexOf( plugin.slug ) === -1;
 }
 
-const PluginsSearchResultPage = ( {
-	search: searchTerm,
+const PluginsSearchResultList = ( {
+	pluginsBySearchTerm,
+	isFetchingPluginsBySearchTerm,
+	pluginsPagination,
+	searchTerm,
```

Comment:
```suggestion
	plugins,
	isFetching,
	pluginsPagination,
	searchTerm,
	categoryName,
```

This might help explain what I'm getting at in the other comment.